orbit/pkg/table/login_password/login_password.go

```diff
@@ -0,0 +1,67 @@
+//go:build darwin
+// +build darwin
+
+package login_password
+
+import (
+	"context"
+	"fmt"
+	"github.com/osquery/osquery-go/plugin/table"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+	"time"
+)
+
+// Columns is the schema of the table.
+func Columns() []table.ColumnDefinition {
+	return []table.ColumnDefinition{
+		table.IntegerColumn("password_hint_enabled"),
+	}
+}
+
+// Generate is called to return the results for the table at query time.
+// Constraints for generating can be retrieved from the queryContext.
+func Generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	uid, gid, err := getConsoleUidGid()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get console user: %w", err)
+	}
+
+	ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
+	defer cancel()
+	cmd := exec.CommandContext(ctx, "dscl", ".", "-list", "/Users", "hint")
+
+	// Run as the current console user (otherwise we get empty results for the root user)
+	cmd.SysProcAttr = &syscall.SysProcAttr{
+		Credential: &syscall.Credential{Uid: uid, Gid: gid},
+	}
+
+	out, err := cmd.Output()
+	if err != nil {
+		return nil, fmt.Errorf("generate failed: %w", err)
+	}
+
+	res := "0"
+	if len(strings.TrimSpace(string(out))) > 0 {
+		res = "1"
+	}
+	return []map[string]string{{"password_hint_enabled": res}}, nil
+}
+
+// getActiveUserGroup gets the uid and gid of the current (or more accurately, most recently logged
+// in) *console* user. In most scenarios this should be the currently logged in user on the system.
+// Note that getting the current user of the Orbit process is typically going to return root and we
+// need the underlying user.
+func getConsoleUidGid() (uid uint32, gid uint32, err error) {
```

Comment:
This is also defined here:
https://github.com/fleetdm/fleet/blob/39db759c8a073c4deb1347d3892de69b54c0219f/orbit/pkg/table/privaterelay/privaterelay.go#L66-L80

We should define this method on a common package (to avoid code duplication).

Reply:
Good catch @lucasmrod 
I'll see if I can find a good place to put this common code.
Please tell me if you have a recomendation.

Reply:
Location looks good.